### ISSUE TITLE
[a11y-defect-1]: ❗️On focus, button text doesn't meet AA compliance for color contrast

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#63534]
### Point of contact

Jamie Klenetsky Fay

### Severity level

1, Launchblocker. Critical. Must be fixed before launch.

### Details

This is an issue on the final page of the form, the "review and submit" screen.

Give the accordion and edit `<button>` keyboard focus, then use spacebar to activate them. The button background color turns dark blue, and the text is still black, which fails color contrast. This only happens when the button has focus already and you're trying to activate it - you've focused on the accordion heading and want to open it, for example. 

Note that this does **not** happen when using enter to activate the buttons, only spacebar.

### Reproduction steps

1. fill out the form - you'll land on the final page, the review and submit screen.
2. navigate using the keyboard
3. give the accordion heading or Edit button focus
4. hold down spacebar. The background color of the button will turn dark blue.

### Proposed solution or next steps

You'll want to compare your implementation of the accordion and button components to those in the DST, and align yours to those. The DST examples have the proper behavior:
- [Accordion component](https://design.va.gov/components/accordion)
- [Button component](https://design.va.gov/components/button/)

### References, articles, or WCAG support

1. [WCAG Success Criteria 1.4.3: Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
2. [VA's standards, and how to test for color contrast](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review#Prepareforanaccessibilitystagingreview-Useofcolorandcolorcontrastuse-of-color)

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [X] Focus
- [ ] Headings
- [X] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by zack** [comment]
This duplicates #64149
